### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#167] [Esercizi - Viaggi/Piazza]
![Es 2](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/bec338d3-66b7-4b36-bf72-ceaeaf15e54e)
($^1$) Vi ricordo che $f^{−1} : B → A$ è definita come segue: preso $b \in B$ sappiamo che esiste $a \in A | f (a) = b$ (perché $f$ è suriettiva); inoltre $a$ è unico dato che $f$ è iniettiva; riassumendo esiste unico $a$ tale che $f (a) = b$ e si pone $f ^{−1}(b) = a$.

**Comment by Elia-Belli** [comment]
- Definizione di $f \circ f^{-1}: B \rightarrow B | f \circ f^{-1} (b) = f (f^{-1}(b)) = f(a) = b$
 Definizione di $id_B: B \rightarrow B | id_B(b) = b$
Siccome $f \circ f^{-1} (b) = id_B(b)$ su tutto il dominio di definizione allora $f \circ f^{-1} = id_B$

- Definizione di $f^ {-1} \circ f: A \rightarrow A | f ^ {-1} \circ f (a) = f^{-1} (f(a)) = f^ {-1}(b) = a$
 Definizione di $id_A: A \rightarrow A | id_A(a) = a$
Siccome $f^ {-1} \circ f (a) = id_A(a)$ su tutto il dominio di definizione allora $f^ {-1} \circ f = id_A$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/4772b709-1aa0-43a1-a367-de197c223201)

**Comment by daromacarlo** [comment]
![Immagine 2023-12-05 192519](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/2e3c8c5f-698e-4d2d-839a-4aa146864e1a)
